### DISCUSSION TITLE
How do display only a field on a content item

### DISCUSSION THREAD

**Discussion by MEllis-Bethany** [OrchardCMS/OrchardCore#16316] [Q&A] [answered]
I would like to display only a field on a content item. So if I'm given a content item, part name, and field name; I want to display the field of that content item and only that field.

The closest I've gotten is calling BuildDisplayAsync on the field display driver, but it never reaches the view.

### Additional Details
**Description**
I have a case content type which has a report content type that is related to the case and is displayed on the same page.

For the case view, there is a header which contains some fields about that case. These fields are determined by a template, using the Templates module. So the fields won't be known until a specific content item is loaded and displayed. I am able to get these fields with no issues.

Also on the page is a list of related reports. And when these reports are displayed on the page, an edit modal is also displayed with it. In this edit modal, I want to display the fields that are displayed in the header of the case.

**Information Available**
When I'm constructing the view for the related item, and specifically the edit modal, I'll have the information needed to get the specific case I am working with. Using the case, I can get a list of fields shown in the template and their corresponding ContentTypePart and ContentPartField definitions.

**Comment by Piedone** [comment]
You can also use [Placement](https://docs.orchardcore.net/en/latest/reference/core/Placement/) to hide everything apart from the shape of that one field. Also see [Shape Tracing](https://github.com/Lombiq/Helpful-Extensions?tab=readme-ov-file#shape-tracing-helpful-extensions) for help.

**Reply by MEllis-Bethany** [reply]
From what I can tell, Placements will not be helpful because this is all very dynamic. I can't create a file because the fields I want to display can be different for each content type and can be changed at any time. I have added more detail to the original post explaining what my current layout it.

**Reply by Piedone** [reply]
From the additional details, I don't see how Placement wouldn't be applicable. If you display shapes for content parts and fields, then you can also hide them (among others) from Placement.

You can also override the whole content item's display with a [Content shape override](https://docs.orchardcore.net/en/latest/reference/modules/Templates/#content-templates) and just display the field's value from there.

**Comment by douwinga** [comment, marked as answer]
I worked with @MEllis-Bethany on this outside of discussions. While placements could have worked, it would have been a maintenance issue in this particular scenario. In the end it basically came down to that he did not realize he already had the specific field shape and just needed to display it in the view via `@DisplayAsync(myFieldShape)` in the view.
